docs/theory/statistics.rst

```diff
@@ -2,9 +2,84 @@
 Statistical issues
 ==================
 
-This document addresses statistical questions to do with Maud, such as:
+This document addresses statistical questions to do with Maud. Specifically, it
+sets out what Maud does from a statistical point of view, what statistical
+assumptions it makes and how these assumptions can be evaluated.
 
- - How does Maud frame the task of learning from multi-omics data as a
-   statistical inference problem?
+Bayesian inference for metabolic networks
+=========================================
 
- - What are some statistical assumptions that Maud makes?
+In general Bayesian inference offers two ways of expressing information
+probabilistically: 'prior' probability distributions over possible values of
+unknown parameters and measurement models or 'likelihood functions' that
+noisily relate parameter values to measurements. Together these determine a
+posterior distribution that synthesises the two sources of information and can
+be used for statistical inference.
+
+Maud's parameters are as follows:
+
+- thermodynamic parameters (the formation energy for each compound in the
+  network)
+- kinetic parameters governing the behaviour of each reaction under fixed
+  conditions
+- concentrations of unbalanced metabolites (i.e. metabolites whose
+  concentrations can be consumed or produced when the system is in a steady
+  state)
+- concentrations of enzymes
+
+The prior distributions for formation energies come from an upstream analysis
+involving in vitro measurements and inference about the composition of each
+compound. Kinetic parameter priors are inferred from online databases using the
+'respectful modelling' paradigm. Priors for concentrations of unbalanced
+metabolites and enzyme concentrations are determined respectively by
+metabolomics and proteomics measurements where these are available and by
+reasonable physical limits otherwise.
+
+The numbers that Maud treats as measurements are metabolomics data relating to
+balanced metabolites and fluxomics data meausuring the steady-state rates of
+reactions. Balanced metabolite concentration measurements are treated as draws
+from a log-normal distribution with known standard deviation parameter and mean
+parameter :math:`\log(true\ concentration)`. Flux measurements are similarly
+treated as draws from a normal distribution with known standard deviation and
+mean :math:`true flux`.
+
+Overall Maud is fairly straightforward from a statistical point of view. The
+main difficulties are (a) specifying how parameter values determine true steady
+state concentrations and fluxes and (b) making the resulting problem
+computationally tractable despite the need to solve a large system of
+non-linear equations.
+
+Statistical assumptions
+=======================
+
+It is somewhat unusual to use metabolomics data to specify prior distributions
+directly rather than adding an extra measurement model. This choice was made
```

Comment:
In house, this is something that we can consider though. Given a number of technical replicates we can fit these measurements as a calibration model parallel and in series to our model. Michael Betancourt's "falling in love with principled modelling" has a nice example of this.